modules/compute/gke-node-pool/main.tf

```diff
@@ -253,3 +253,62 @@ resource "google_project_iam_member" "node_service_account_artifact_registry" {
   role    = "roles/artifactregistry.reader"
   member  = "serviceAccount:${local.sa_email}"
 }
+
+data "google_container_cluster" "gke_cluster" {
+  project  = var.project_id
+  name     = local.split_cluster_id[5]
+  location = local.split_cluster_id[3]
+}
```

Comment:
This data lookup does not seem strictly needed. Also does not look like it is hurting anything. What is your rational for adding?

Reply:
removed